### PR TITLE
test_resources_infra

### PR DESCRIPTION
**Pull request by rrasouli**
Checking the CPU and memory resources after creating and refreshing infrastructure provider

### CODE REVIEW

**Comment by psav** [comment]
ipython and pdfminer are required for our tests

**Comment by dajoRH** [comment]
Lint report for commit d6bc69cf973dd86f143c58c22abe34d8a47f1ca3:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by rrasouli** [comment]
psav: done I uncommented them

**Comment by dajoRH** [comment]
Lint report for commit 205319fefa14a12d26e18fd9e4b9c0383e1504a7:

`cfme/tests/infrastructure/test_resources.py`:
- :red_circle: Line 4:1: **F401** *'Quadicon' imported but unused*
- :red_circle: Line 4:1: **F401** *'toolbar' imported but unused*
- :red_circle: Line 5:1: **F401** *'Host' imported but unused*
- :red_circle: Line 6:1: **F401** *'InfoBlock' imported but unused*
- :red_circle: Line 7:1: **F401** *'sel' imported but unused*
- :red_circle: Line 8:1: **F401** *'is_host_analysis_finished' imported but unused*
- :red_circle: Line 9:1: **F401** *'wait_for' imported but unused*
- :bangbang: Line 15:101: **E501** *line too long (104 > 100 characters)*
- :bangbang: Line 18:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 20:1: **E304** *blank lines found after function decorator*
- :red_circle: Line 22:21: **F821** *undefined name 'get_detail'*
- :bangbang: Line 22:101: **E501** *line too long (119 > 100 characters)*
- :red_circle: Line 23:21: **F821** *undefined name 'get_detail'*
- :red_circle: Line 24:21: **F821** *undefined name 'get_detail'*
- :bangbang: Line 24:101: **E501** *line too long (112 > 100 characters)*
- :bangbang: Line 26:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 27:6: **E111** *indentation is not a multiple of four*
- :bangbang: Line 28:6: **E111** *indentation is not a multiple of four*
- :red_circle: Line 28:22: **F821** *undefined name 'get_detail'*
- :bangbang: Line 28:101: **E501** *line too long (106 > 100 characters)*
- :heavy_exclamation_mark: Line 28:107: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by psav** [comment]
Will this PR be fixed?

**Comment by rrasouli** [comment]
Fixed pep8 and flake8 issues, and missing object before get_detail - should be fine now

**Comment by dajoRH** [comment]
Lint report for commit 92d3ae5651b25cfcbb74d0b5a029634aeb82ae8c:

`cfme/tests/infrastructure/test_resources.py`:
- :bangbang: Line 26:17: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@rrasouli it is not :)

**Comment by dajoRH** [comment]
Lint report for commit b053d154ca6686d98253d05d97d71180337663c4:

`cfme/tests/infrastructure/test_resources.py`:
- :bangbang: Line 26:17: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 92b4978ebe1c819250dd4500722e01007793da21:

`cfme/tests/infrastructure/test_resources.py`:
- :heavy_exclamation_mark: Line 27:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_resources.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import pytest
+from utils import testgen
+
+pytestmark = [pytest.mark.meta(server_roles='+ smartproxy + smartstate')]
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_resources.py]: It must be `'+smartproxy +smartstate'`

**Comment by psav** [comment]
@rrasouli please can you review this PR and close it if no longer necessary, we have too many open, stale PRs right now

**Comment by rrasouli** [comment]
closing this PR as opened new one later

**Comment by rrasouli** [comment]
Closed

Thanks,

Ronnie Rasouli
Quality Engineer RHOS CloudForms
Red Hat

**Pull request closed without merging**